### PR TITLE
chore: add the configuration item maxLineSize

### PR DESCRIPTION
**Pull request by gueFDF**
FIX : https://github.com/openGemini/openGemini/issues/551
### What problem does this PR solve?

Issue Number: [#551 
](https://github.com/openGemini/openGemini/issues/551)
chore: add the configuration item maxLineSize
### What is changed and how it works?
Add a maxLineSize to handle configuration items

### How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [ ] Test A
- [ ] Test B
- [ ] Test cases to be added
- [ ] No code

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: config/openGemini.singlenode.conf [hunk 1]
```diff
@@ -21,6 +21,7 @@
   store-wal-dir = "/tmp/openGemini/data"
   store-meta-dir = "/tmp/openGemini/meta"
   enable-mmap-read = false
+  max-line-size = 65536
```
Comment (xiangyu5632) [inline config/openGemini.singlenode.conf:24]: also need to add `max-line-size` in openGemini.conf

File: lib/util/lifted/influx/httpd/handler.go [hunk 2]
```diff
@@ -1247,7 +1247,7 @@ func (h *Handler) serveWrite(w http.ResponseWriter, r *http.Request, user meta2.
 		tsMultiplier = 1e9 * 3600
 	}
 
-	ctx := influx.GetStreamContext(body)
+	ctx := influx.GetStreamContext(body,h.Config.MaxLineSize)
```
Comment (xiangyu5632) [inline lib/util/lifted/influx/httpd/handler.go:1250]: we need to add UT for GetStreamContext, because a new parameter added

Comment (gueFDF) [reply lib/util/lifted/influx/httpd/handler.go]: added

**Comment by xiangyu5632** [comment]
@gueFDF pls resolve conflicts

File: config/openGemini.singlenode.conf [hunk 3]
```diff
@@ -13,6 +13,7 @@
   flight-address = "127.0.0.1:8087"
   flight-enabled = true
   flight-auth-enabled = false
+  max-line-size = 65536
```
Comment (shilinlee) [inline config/openGemini.singlenode.conf:16]: Don't need to define `max-line-size = 65536` for the default behavior. Just remove it please.

**Review by shilinlee — CHANGES_REQUESTED** [review]

**Review by xiangyu5632 — APPROVED** [review]

**Review by shilinlee — APPROVED** [review]

File: config/openGemini.conf
Comment (shilinlee) [inline config/openGemini.conf:1]: Where is the difference?

**Review by shilinlee — CHANGES_REQUESTED** [review]

File: config/openGemini.conf
Comment (shilinlee) [reply config/openGemini.conf:1]: Could you please solve the meanless whitespace diff?

**Review by shilinlee — CHANGES_REQUESTED** [review]

File: config/openGemini.conf
Comment (shilinlee) [reply config/openGemini.conf:1]: Still exist like:

image

**Pull request closed without merging**